mapserver/README.md

```diff
@@ -13,15 +13,15 @@ This will start a mapserver container and a mapcache container, mapped to ports
 
 ### Structure
 
-Mapcache will receive all tile requests from browser clients in `WMS`. Though mapcache can also accept requests in the more used `TMS` or `gmaps` formats, it only supports _dimensions_ in WMS. The service is configured to accept a `DATAFILE` dimension, which is used to determine the source raster so clients can request a specific unsatisfied demand raster, falling back to `KEN_popmap15_v2b` that contains the original demographics.
+Mapcache will receive all tile requests from browser clients in `WMS`. Though mapcache can also accept requests in the more used `TMS` or `gmaps` formats, it only supports _dimensions_ in WMS. The service is configured to accept a `DATAFILE` dimension, which is used to determine the source raster so clients can request a specific unsatisfied demand raster.
 
 Mapcache will forward all requests to Mapserver in WMS format as well, and cache the requested tiles locally on disk at `/tmp`. Mapserver is configured to respond both in tile mode and `WMS`, and accepts the `DATAFILE` query param to change the source raster.
 
 ### Endpoints
 
-* To query mapcache in WMS format the root URL should be `http://localhost:5002/mapcache`, specifying `kenya` as `LAYER`, and optionally setting `DATAFILE`. For example:
+* To query mapcache in WMS format the root URL should be `http://localhost:5002/mapcache`, specifying `population` as `LAYER`, and setting `DATAFILE`. For example:
 ```
-http://localhost:5002/mapcache?&SERVICE=WMS&REQUEST=GetMap&VERSION=1.1.1&LAYERS=kenya&STYLES=&FORMAT=image%2Fjpeg&TRANSPARENT=true&HEIGHT=256&WIDTH=256&DATAFILE=KEN_popmap15_v2b&SRS=EPSG%3A3857&BBOX=3913575.848201024,-313086.067856082,4070118.882129065,-156543.03392804056
+http://localhost:5002/mapcache?&SERVICE=WMS&REQUEST=GetMap&VERSION=1.1.1&LAYERS=population&STYLES=&FORMAT=image%2Fjpeg&TRANSPARENT=true&HEIGHT=256&WIDTH=256&DATAFILE=KEN_popmap15_v2b&SRS=EPSG%3A3857&BBOX=3913575.848201024,-313086.067856082,4070118.882129065,-156543.03392804056
 ```
 
 * To query mapcache in gmaps format, which does not support the `DATAFILE` parameter:
```

Comment:
Does it still make sense to query other services other than WMS since the `DATAFILE` dimension is now required?